src/types/zenvia.ts

```diff
@@ -163,3 +163,18 @@ export interface IWebhook {
     [header: string]: string;
   };
 }
+
+export interface IChannels {
+  type: 'WHATSAPP' | 'FACEBOOK' | 'SMS';
```

Comment:
mmm... I know in the API is like this already, so perhaps is too late to change this, but perharps we should create a new version where this is consistent with the type Channel.

What you guys think? @wmangelardo @hmagarotto @rodrigokamada